ppl/src/test/java/org/opensearch/sql/ppl/parser/AstExpressionBuilderTest.java

```diff
@@ -149,6 +149,38 @@ public void testLogicalLikeExpr() {
         filter(relation("t"), function("like", field("a"), stringLiteral("_a%b%c_d_"))));
   }
 
+  @Test
+  public void testLikeOperatorExpr() {
+    // Test LIKE operator syntax
+    assertEqual(
+        "source=t a LIKE '_a%b%c_d_'",
```

Comment:
as we plan to improve search command to support full-text search, how can we differenate a is field or keyword user want to search?

Reply:
Good catch. When we change the grammar in the upcoming PR for free text grammar, this test will fall.
I will proactively remove now.

Reply:
Changed tests.